otter/grade/__init__.py

```diff
@@ -43,6 +44,7 @@ def main(
     timeout: bool = None,
     no_network: bool = False,
     debug: bool = False,
+    result_queue: Queue = None,
```

Comment:
type annotation should be `Optional[Queue]`

Reply:
I got this change in